src/client/js/LinkUrlDialog.js

```diff
@@ -97,7 +98,7 @@ export default class LinkUrlDialog extends React.Component {
                           id="linkTitle"
                           style={styles.linkTitleInput}
                           type="text"
-                          value={this.props.titleValue}
+                          value={this.state.titleValue}
```

Comment:
LinkUrlDialog should use own state for editor variables, not use external props/